src/components/AppIcon/icons/LogoIcon.tsx

```diff
@@ -0,0 +1,26 @@
+const LogoIcon = () => {
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 36 36">
```

Comment:
`<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 36 36" {...rest}>`

Reply:
Could be.

But you will not be able to change lots of settings via   `<svg  props...`

In some projects we pass `{color}` and even {`backColor, backColor} `but you need to hack content of SVG, `<path fill={backColor} ... `` <line stroke={backColor} and so on`